test/recipes/15-test_genrsa.t

```diff
@@ -101,6 +101,9 @@ ok(!run(app([ 'openssl', 'genpkey', '-algorithm', 'RSA',
              '-pkeyopt', 'e:65538',
              '-out', 'genrsatest.pem' ])),
    "genpkey with a even public exponent should fail");
+ok(!run(app([ 'openssl', 'genpkey', '-propquery', 'unknown',
+             '-algorithm', 'RSA' ])),
+   "genpkey using unknown provider should fail");
```

Comment:
Specifying `unknown` like this actually does not mean to ask for unknown provider but for an `unknown=yes` property to be set on the `RSA` algorithm.

So the message should be something like:
`genpkey requesting unknown=yes property should fail`

Reply:
Oh, you are right!